packages/plugins/whisper/src/index.js

```diff
@@ -38,6 +50,57 @@ class Whisper {
     });
   }
 
+  shouldInit() {
+    return (
+      this.communicationConfig.enabled
+    );
+  }
+
+  _getNodeState(err, version, cb) {
+    if (err) return cb({ name: "Ethereum node not found", status: 'off' });
+
+    let nodeName = "go-ethereum";
+    let versionNumber = version.split("-")[0];
+    let name = nodeName + " " + versionNumber + " (Ethereum)";
+    return cb({ name, status: 'on' });
+  }
+
+  // TODO: need to get correct port taking into account the proxy
+  registerServiceCheck() {
+    this.events.request("services:register", 'Ethereum', (cb) => {
+      const { rpcHost, rpcPort, wsRPC, wsHost, wsPort } = this.blockchainConfig;
+      if (wsRPC) {
+        return ws(wsHost, wsPort + 10, (err, version) => this._getNodeState(err, version, cb));
+      }
+      rpc(rpcHost, rpcPort + 10, (err, version) => this._getNodeState(err, version, cb));
```

Comment:
No need for the +10, unless we really want to test the proxy? If so, we'd need a request from the proxy component to have the actual port

Reply:
Good point, removed.